src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -2,11 +2,15 @@
 
 {% load staticfiles %}
 
+{% block metatitle %}
+    <title>Model My Watershed - Micro Site Storm Model</title>
+{% endblock metatitle %}
+
 {% block header %}
     <header class="water-balance">
 
         <nav id="app-header" class="navbar-watershed">
-            <div class="brand"><a href="#">Model My Watershed</a></div>
+            <div class="brand"><a href="#">Model My Watershed - Micro Site Storm Model</a></div>
```

Comment:
Actually, yeah - they really want to emphasize the relationship between these two apps, and we'll need to put some thoughts towards that (i've got some ideas), but for now, at least having basic navigation is a good step.  I don't think we need the second link for the micro site though, we don't have anything to wire up to it.